superset/utils/core.py

```diff
@@ -1022,6 +1022,13 @@ def get_example_database() -> "Database":
     return get_or_create_db("examples", db_uri)
 
 
+def get_main_database() -> "Database":
```

Comment:
We usually refer to this as the "metadata" database - should that be in the function name?

Reply:
it is called in superset as a main database on the dev installation, I am fine with renaming it - however we should probably do it across the board in a separate PR

Reply:
Ah, forgive the confusion on my part. This is fine as-is then.